cve_bin_tool/checkers/gcc.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for gcc
+
+https://www.cvedetails.com/vulnerability-list/vendor_id-72/product_id-960/GNU-GCC.html
+
+"""
+from . import Checker
+
+
+class GccChecker(Checker):
+    CONTAINS_PATTERNS = [
+        r"gcc: fatal error: no input files",
+        r"compilation terminated.",
+    ]
+    FILENAME_PATTTERN = [r"gcc"]
+    VERSION_PATTERNS = [r"GCC ([0-9]+\.[0-9]+\.[0-9]+)", r"GCC ([0-9]+\.[0-9]+)"]
+    VENDOR_PRODUCT = [
+        ("GCC", "GCC"),
+        ("GNU", "GCC"),
```

Comment:
Try lowercase for these.  It looks like the tests are erroring out because it can't find the product.